### PR TITLE
✅(scripts) add argument to ci-test-route to set the max retries

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

The `ci-test-route`  script has  a hardcoded value  (5) for  the maximum number of retries to get the  expected response from the server before failing. 

This  number  is   too  low  for  the  `test-bootstrap-edxapp` test suite and  generates a lot  of false negative  in the CI.  

## Proposal

- Add a new argument  to the  script to  specify the  maximum number  of retries.

- Increase max_retry to 20 for `test-bootstrap-edxapp` tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- Add argument to `ci-test-route` to configure the maximum number of retries
+
```
Comment (jmaupetit) [inline CHANGELOG.md:14]: As it is related to the project, I'm not sure we should log this... WDYT @lunika?

**Review by jmaupetit — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lunika) [reply CHANGELOG.md:14]: I think we can keep this new entry in the changelog.

Comment (madmatah) [reply CHANGELOG.md:14]: Thanks ! I'll merge this one when the CI checks are finished then.
